pytorch_lightning/trainer/connectors/checkpoint_connector.py

```diff
@@ -56,18 +56,22 @@ def hpc_resume_path(self) -> Optional[str]:
         max_version = self.max_ckpt_version_in_folder(dir_path_hpc, "hpc_ckpt_")
         if max_version is not None:
             return os.path.join(dir_path_hpc, f"hpc_ckpt_{max_version}.ckpt")
-        auto_save_checkpoint = os.path.join(dir_path_hpc, ".pl_auto_save.ckpt")
-        if os.path.exists(auto_save_checkpoint):
-            return auto_save_checkpoint
+
+    @property
+    def fault_tolerant_auto_save_path(self) -> Optional[str]:
```

Comment:
Make this protected?

Reply:
👍 make `_hpc_resume_path` too?

Reply:
hmm, but that was already public

Reply:
hm yea, either protected or not I was thinking they should be consistent? (same auto-resume path concepts)

Reply:
on a related note, rename `fault_tolerant_auto_save_path` --> `fault_tolerant_auto_resume_path` ?

Reply:
> hmm, but that was already public

@awaelchli would you say we should honor deprecation for public properties in connectors? (not mixins).
Wouldn't you consider them private as they are completely hidden from the user API?
One of their downsides is that they limit customization of the Trainer, which also means it's unlikely any users will have customized them.

If you agree with this and we don't deprecate, resolve this conversation so the PR gets merged :P

Reply:
sorry for the delay, I didn't realize this depends on me.